BotDatabase.py

```diff
@@ -19,16 +26,28 @@ def __del__(self):
 
     def Open(self):
         self.Close()
-        self.Database = sqlite3.connect(Config.GetDBFile())
+        ConfigData=Config()
```

Comment:
There's no need to initialize this, bringing in from the import, means that you can just use the static methods easily.

Reply:
Artifact from code testing in isolation. Will remove